client/config/environment.js

```diff
@@ -65,10 +64,10 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' 'https://www.google.com' 'www.google-analytics.com'",
```

Comment:
Pretty sure we still need that but I can test it out again when I get back to my desk